packages/components/src/scroll-lock/index.js

```diff
@@ -15,7 +15,7 @@ import { Component } from '@wordpress/element';
  * @return {Component} The bound ScrollLock component.
  */
 export function createScrollLockComponent( {
-	htmlDocument = document,
+	htmlDocument = typeof document !== 'undefined' ? document : undefined,
```

Comment:
This one is only required because we create a "default" scroll lock component at the end of the file. Perhaps we could create it on-demand instead or creating it during module init?